etna/collections/models.py

```diff
@@ -260,6 +260,7 @@ class TimePeriodExplorerPage(AlertMixin, BasePageWithIntro):
         FieldPanel("body"),
         FieldPanel("start_year"),
         FieldPanel("end_year"),
+        InlinePanel("page_time_periods", label="Time Period Pages")
```

Comment:
Note: @ababic / @jamesbiggs - I'm not sure if this was intended to be left out?

Reply:
The `page_time_periods` InlinePanel is used for "tagging" the pages/relating them to a `TimePeriodExplorerPage` (e.g. Medieval, Post-Modern, etc). This page model is what the InlinePanel would be referring to, so I don't think we want it on there because we don't need to "tag" the page

Reply:
Right but how do these tags show up in related listing if we aren't building at least a few relationships between them?

Reply:
The `page_time_periods` works by giving the `TimePeriodPage` a reference to the "parent" `TimePeriodExplorerPage`. Then we use the `.filter(pk__in=self.related_page_pks)` to get all the `TimePeriodPage`s that contain a reference to the `TimePeriodExplorerPage`. `page_time_periods` would only allow the `TimePeriodExplorerPage` to reference itself (or another `TimePeriodExplorerPage`) which isn't what we want here